pkg/kubelet/stats/cadvisor_stats_provider_test.go

```diff
@@ -225,7 +225,7 @@ func TestCadvisorListPodStats(t *testing.T) {
 	con := indexCon[cName00]
 	assert.EqualValues(t, testTime(creationTime, seedPod0Container0).Unix(), con.StartTime.Time.Unix())
 	checkCPUStats(t, "Pod0Container0", seedPod0Container0, con.CPU)
-	checkMemoryStats(t, "Pod0Conainer0", seedPod0Container0, infos["/pod0-c0"], con.Memory)
+	checkMemoryStats(t, "Pod0Container0", seedPod0Container0, infos["/pod0-c0"], con.Memory)
```

Comment:
(confirmed that this is just a label used for printing test output, so the fact that it was wrong before doesn't mean the test was broken.)